### PR TITLE
adding kibana_server role specification and explanation #4094

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Adding specification of kibana_server role with explanation of the permissions assigned and mapping needed

### Issues Resolved
Closes https://github.com/opensearch-project/documentation-website/issues/4094

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks, @AntonEliatra!

**Review by scrawfor99 — APPROVED** [review]

**Comment by hdhalter** [comment]
Thanks, @scrawfor99!

File: _security/multi-tenancy/multi-tenancy-config.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ config:
 | `multitenancy_enabled` | Enable or disable multi-tenancy. Default is `true`. |
 | `private_tenant_enabled` | Enable or disable the private tenant. Default is `true`. |
 | `default_tenant` | Use to set the tenant that is available when users log in. |
-| `server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`. |
+| `server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`. If a different user is configured, make sure this user is mapped to `kibana_server` role via `role_mappings.yml` file, to give the appropriate permissions listed in [kibana_server role details]({{site.url}}{{site.baseurl}}/security/multi-tenancy/multi-tenancy-config/#kibana_server-role-details) |
```
Comment (vagimeli) [inline _security/multi-tenancy/multi-tenancy-config.md:30]:
```suggestion
| `server_username` | Must match the name of the OpenSearch Dashboards server user from `opensearch_dashboards.yml`. Default is `kibanaserver`. If a different user is configured, then make sure such user is mapped to the `kibana_server` role via the `role_mappings.yml` file to give that user the appropriate permissions listed in [kibana_server role details]({{site.url}}{{site.baseurl}}/security/multi-tenancy/multi-tenancy-config/#kibana_server-role-details). |
```

File: _security/multi-tenancy/multi-tenancy-config.md [hunk 2]
```diff
@@ -149,3 +149,93 @@ The Security plugin scrubs these index names of special characters, so they migh
 {: .tip }
 
 To back up your OpenSearch Dashboards data, [take a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/) of all tenant indexes using an index pattern such as `.kibana*`.
+
+## `kibana_server` role details
+
+`kibana_server` role is used by OpenSearch Dashboards to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role via `role_mappings.yml`. You can view the full list of permissions assigned to this role by running `GET` request on `_plugins/_security/api/roles/kibana_server` API using admin certificate, key and certificate authority file.
```
Comment (vagimeli) [inline _security/multi-tenancy/multi-tenancy-config.md:155]:
```suggestion
OpenSearch Dashboards uses the`kibana_server` role to perform necessary operations on OpenSearch. By default `kibanauser` is mapped to this role through the `role_mappings.yml` file. You can view the full list of permissions assigned to this role by running the GET request on `_plugins/_security/api/roles/kibana_server` API using the admin certificate, key, and certificate authority file.
```

File: _security/multi-tenancy/multi-tenancy-config.md [hunk 3]
```diff
@@ -149,3 +149,93 @@ The Security plugin scrubs these index names of special characters, so they migh
...
+Following is the full list of permissions assigned to this role:
```
Comment (vagimeli) [inline _security/multi-tenancy/multi-tenancy-config.md:156]:
```suggestion
The following list includes the permissions assigned to this role:
```

**Review by vagimeli — APPROVED** [review]
Doc review is complete. Minimal changes.

File: _security/multi-tenancy/multi-tenancy-config.md (hunk 1 above)
Comment (natebower) [reply _security/multi-tenancy/multi-tenancy-config.md:30]: Edited suggestion

File: _security/multi-tenancy/multi-tenancy-config.md [hunk 4]
```diff
@@ -149,3 +149,93 @@ The Security plugin scrubs these index names of special characters, so they migh
 {: .tip }
 
 To back up your OpenSearch Dashboards data, [take a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/) of all tenant indexes using an index pattern such as `.kibana*`.
+
+## `kibana_server` role details
```
Comment (natebower) [inline _security/multi-tenancy/multi-tenancy-config.md:153]:
```suggestion
## `kibana_server` role
```

File: _security/multi-tenancy/multi-tenancy-config.md (hunk 2 above)
Comment (natebower) [reply _security/multi-tenancy/multi-tenancy-config.md:155]: Edited suggestion. Is "on the...API" the correct wording? The last sentence is a bit of a run-on.

**Review by natebower — COMMENTED** [review]
@AntonEliatra @vagimeli I edited the suggestions still present in the file and added a comment. Let me know if you have any questions. Thanks!

**Comment by AntonEliatra** [comment]
@natebower Thanks for feedback, I've updated to :
You can view the full list of permissions assigned to this role by sending a GET request to the `_plugins/_security/api/roles/kibana_server` API (include the admin certificate, key, and certificate authority file in the GET request).

**Comment by natebower** [comment]
> @natebower Thanks for feedback, I've updated to : You can view the full list of permissions assigned to this role by sending a GET request to the `_plugins/_security/api/roles/kibana_server` API (include the admin certificate, key, and certificate authority file in the GET request).

Works for me, thanks!

**Pull request merged**